tasks/README.md

```diff
@@ -377,4 +380,4 @@ Name | Summary | Category
 `task518_emo_different_dialogue_emotions` | Given different dialogue determine if they have the same emotion | Classification, Sentiment Analysis
 `task519_aquamuse_question_generation` | Given an answer generate a question that would be answered by the answer given | Question Generation
 `task520_aquamuse_answer_given_in_passage` | Given a passage and a question determine if the question can be answered by the passage | Classification
-
+`task520_aquamuse_answer_given_in_passage` | Given a passage and a question determine if the question can be answered by the passage | Classification
```

Comment:
I am not sure if the merge is done correctly. There are a bunch of new lines here.

Reply:
> I am not sure if the merge is done correctly. There are a bunch of new lines here.

I removed that line.